packages/model/src/user.ts

```diff
@@ -100,13 +100,46 @@ export interface UserDataObject extends Invitee {
   workingGroups: WorkingGroupMembership[];
   interestGroups: InterestGroupMembership[];
 }
-export type ListUserDataObject = ListResponse<UserDataObject>;
+
 export interface UserResponse
   extends Omit<UserDataObject, 'onboarded' | 'connections'> {
   onboarded: boolean;
   displayName: string;
 }
 
+export type UserListItemTeam = Pick<UserTeam, 'id' | 'displayName' | 'role'>;
+export type UserListItemDataObject = Pick<
+  UserResponse,
+  | 'alumniSinceDate'
+  | 'avatarUrl'
+  | 'city'
+  | 'country'
+  | 'createdDate'
+  | 'degree'
+  | 'dismissedGettingStarted'
+  | 'displayName'
+  | 'email'
+  | 'firstName'
+  | 'id'
+  | 'institution'
+  | 'jobTitle'
+  | 'labs'
+  | 'lastName'
+  | 'membershipStatus'
+  | 'onboarded'
+  | 'role'
+> & {
+  _tags: string[];
+  teams: UserListItemTeam[];
+};
+export type ListUserDataObject = ListResponse<UserListItemDataObject>;
+
+export type UserListItemResponse = UserListItemDataObject;
+
+export type UserAlgoliaResponse = UserListItemDataObject;
+
+export type UserListAlgoliaResponse = ListResponse<UserAlgoliaResponse>;
```

Comment:
I think we still don't want to expose engine specific response types. Check out how this was done for Teams:

https://github.com/yldio/asap-hub/blob/91758e7b2438325d6bb579f6b23cbaf9724f0750/packages/model/src/team.ts#L86

btw you were the one to propse it. It might sound like just a naming convention but it really isn't.

Reply:
I've renamed it here [review: rename things with Algolia to UserItem](https://github.com/yldio/asap-hub/pull/4020/commits/22969e0ea8949b58dbba01630b2c9f591a51e206) + [fix rename mistake](https://github.com/yldio/asap-hub/pull/4020/commits/333767be894dce993026378c49ba9021305b93d9)